tools/bgpreader.c

```diff
@@ -293,8 +314,10 @@ int main(int argc, char *argv[])
   int k;
   for (k = 0; k < OPTIONS_CNT; k++) {
     size_t size = strlen(short_options);
-    snprintf(short_options + size, sizeof(short_options) - size,
-             OPTIONS[k].option.has_arg ? "%c:": "%c", OPTIONS[k].option.val);
+    if(isalpha(OPTIONS[k].option.val)) {
+      snprintf(short_options + size, sizeof(short_options) - size,
+               OPTIONS[k].option.has_arg ? "%c:": "%c", OPTIONS[k].option.val);
+    }
```

Comment:
@salsh missing a `else` here?

Reply:
The "else" part is empty because the non-alpha options (long args) are included in ```long_options``` in every iteration anyway (short args also have long args but the RPKI related long args do not have short args).